.github/workflows/main.yml

```diff
@@ -38,21 +38,16 @@ jobs:
           restore-keys: |
             0-cache-macos-latest
         if: matrix.os == 'macos-latest'
-      - name: Install wasmtime for tests
-        # TODO: switch to Wasmtime 17 once it's released, which will include https://github.com/bytecodealliance/wasmtime/pull/7750
-        run: |
-          curl -f -L --retry 5 https://wasmtime.dev/install.sh | bash -s -- --version dev
-          ~/.wasmtime/bin/wasmtime --version
-          curl -f -L --retry 5  -o ~/.wasmtime/bin/wasi_snapshot_preview1.command.wasm https://github.com/bytecodealliance/wasmtime/releases/download/dev/wasi_snapshot_preview1.command.wasm
-          if [ "${{ matrix.os }}" == "ubuntu-latest" ]; then
-          curl -f -OL --retry 5 https://github.com/bytecodealliance/wasm-tools/releases/download/wasm-tools-1.0.54/wasm-tools-1.0.54-x86_64-linux.tar.gz
-          tar xf wasm-tools-1.0.54-x86_64-linux.tar.gz
-          cp wasm-tools-1.0.54-x86_64-linux/wasm-tools ~/.wasmtime/bin/
-          else
-          curl -f -OL --retry 5 https://github.com/bytecodealliance/wasm-tools/releases/download/wasm-tools-1.0.54/wasm-tools-1.0.54-x86_64-macos.tar.gz
-          tar xf wasm-tools-1.0.54-x86_64-macos.tar.gz
-          cp wasm-tools-1.0.54-x86_64-macos/wasm-tools ~/.wasmtime/bin/          
-          fi
+      - name: Setup `wasmtime`
```

Comment:
Maybe keep the "for tests" part of the old title, or include it in a comment so its clear that these things are only used for testing and not part of wasi-sdk itself.

Reply:
Sure, updated.